### PR TITLE
Stories style

### PR DESCRIPTION
**Pull request by RobStallion**
create and style stories page #20 
tidy app.js to make code more readable 
Create stories module to allow users to send testimonial emails #20

### CODE REVIEW

File: lib/modules/apostrophe-assets/public/js/site.js [hunk 1]
```diff
@@ -2,4 +2,27 @@ $(function() {
...
+          readMoreBtn.text("Read less")
+      } else {
+          singleWrapperDiv.addClass("max-h")
+          readMoreBtn.text("Read more")
+        }
+      });
+    } else {
+      readMoreBtn.addClass("dn")
+    }
+  })
```
Comment (samhstn) [inline lib/modules/apostrophe-assets/public/js/site.js:27]: This is pretty nice 👍

File: lib/modules/messages-submit-widgets/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      };
+      var detailsForSendMany = {
+        templateName: 'template',
+        context: person,
+        subject: person.subject,
+        toAddresses: [person.email],
+        senderEmailAddress: person.senderEmailAddress,
+        replyToAddress: person.replyToAddress,
+        ccAddresses: null,
+        bccAddresses: null
```
Comment (samhstn) [inline lib/modules/messages-submit-widgets/index.js:25]: If we get this `sendemail` pr merged: https://github.com/dwyl/sendemail/pull/87, then we can clean up some of this code

**Review by samhstn — CHANGES_REQUESTED** [review]
Reassign once merge conflicts are fixed. Also, getting this merged could improve the `sendemail` logic, https://github.com/dwyl/sendemail/pull/87

File: lib/modules/messages-submit-widgets/index.js (hunk 2 above)
Comment (RobStallion) [reply lib/modules/messages-submit-widgets/index.js:25]: I think that would be best. I have reviewed [dwyl/sendemail#87](https://github.com/dwyl/sendemail/pull/87) now. Do you want to hold off on merging this until that has been merged or shall we merge this and refactor it later?

Comment (samhstn) [reply lib/modules/messages-submit-widgets/index.js:25]: I think we should merge this pr in, then we can make the `sendMany -> email` change when `sendemail` has been updated

**Review by samhstn — APPROVED** [review]
Looks good 👍, @parkhabit merge whenever you're happy with the code

File: lib/modules/messages-submit-widgets/index.js (hunk 2 above)
Comment (RobStallion) [reply lib/modules/messages-submit-widgets/index.js:25]: Sounds good

**Review by parkhabit — APPROVED** [review]
🏆 awesome stuff

**Pull request merged**